src/panoptes/pocs/camera/camera.py

```diff
@@ -921,20 +927,53 @@ def _setup_observation(self, observation, headers, filename, **kwargs):
             metadata['filter_request'] = observation.filter_name
 
         if headers is not None:
+            self.logger.trace(f'Updating {file_path} metadata with provided headers')
             metadata.update(headers)
 
-        self.logger.debug(
-            f'Observation setup: exptime={exptime} file_path={file_path} image_id={image_id} '
-            f'metadata={metadata}')
+        self.logger.debug(f'Observation setup: {exptime=} {file_path=} {image_id=} {metadata=}')
+
         return exptime, file_path, image_id, metadata
 
-    def _process_fits(self, file_path, info):
+    def _process_fits(self, file_path, metadata):
```

Comment:
If the only purpose of this function is to add the fits headers, I suggest to rename to `_add_fits_headers`.

Reply:
All of the concrete camera classes are supposed to override this and they do more elaborate things than just adding FITS (potentially at least).

Agree that the name and structure need to change but that's part of the larger refactoring @AnthonyHorton and I always want to get to.